src/cli/parser.rs

```diff
@@ -287,11 +383,20 @@ fn parse_rgb() {
     assert_eq!(Some(rgb(255, 0, 119)), parse_color("255,0,119"));
     assert_eq!(Some(rgb(255, 0, 119)), parse_color("  255  ,  0  ,  119  "));
     assert_eq!(Some(rgb(1, 2, 3)), parse_color("1,2,3"));
+    assert_eq!(Some(rgb(255, 0, 127)), parse_color("rgb(100%,0%,49.8%)"));
+    assert_eq!(Some(rgb(255, 0, 153)), parse_color("rgb(100%,0%,60%)"));
+    assert_eq!(Some(rgb(255, 0, 119)), parse_color("rgb(100%,0%,46.7%)"));
+    assert_eq!(Some(rgb(3, 54, 119)), parse_color("rgb(1%,21.2%,46.7%)"));
+    assert_eq!(Some(rgb(255, 0, 119)), parse_color("rgb(255 0 119)"));
+    assert_eq!(Some(rgb(255, 0, 119)), parse_color("rgb(    255      0      119)"));
 
     assert_eq!(None, parse_color("rgb(256,0,0)"));
     assert_eq!(None, parse_color("rgb(255,0)"));
     assert_eq!(None, parse_color("rgb(255,0,0"));
     assert_eq!(None, parse_color("rgb (256,0,0)"));
+    assert_eq!(None, parse_color("rgb (250%,240%,180%)"));
+    assert_eq!(None, parse_color("rgb (100%,0,0)"));
+    assert_eq!(None, parse_color("rgb (100%,555%,-45%)"));
```

Comment:
These three are not parsed as a number due to the space between `rgb` and `(`. However, it looks like you wanted to add tests for something else (too high or negative percentages?)

Reply:
Yes I made some errors when I pasted :confused:.